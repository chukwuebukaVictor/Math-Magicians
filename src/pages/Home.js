import React from 'react';

function Home() {
  return (
    <>
      <div className="welcome">
        <h1>Welcome to Our Page</h1>
        <p>
          Lorem Ipsum è un testo segnaposto utilizzato nel
          settore della tipografia e della stampa. Lorem Ipsum
          è considerato il testo segnaposto standard sin dal
          sedicesimo secolo, quando un anonimo tipografo prese
          una cassetta di caratteri e li assemblò per preparare
          un testo campione. È sopravvissuto non solo a più di
          cinque secoli, ma anche al passaggio alla
          videoimpaginazione, pervenendoci sostanzialmente inalterato.
        </p>
        <p>
          Lorem Ipsum è un testo segnaposto utilizzato nel
          settore della tipografia e della stampa. Lorem Ipsum
          è considerato il testo segnaposto standard sin dal
          sedicesimo secolo, quando un anonimo tipografo prese
          una cassetta di caratteri e li assemblò per preparare
          un testo campione. È sopravvissuto non solo a più di
          cinque secoli, ma anche al passaggio alla
          videoimpaginazione, pervenendoci sostanzialmente inalterato.
        </p>
        <p>
          Lorem Ipsum è un testo segnaposto utilizzato nel
          settore della tipografia e della stampa. Lorem Ipsum
          è considerato il testo segnaposto standard sin dal
          sedicesimo secolo, quando un anonimo tipografo prese
          una cassetta di caratteri e li assemblò per preparare
          un testo campione. È sopravvissuto non solo a più di
          cinque secoli, ma anche al passaggio alla
          videoimpaginazione, pervenendoci sostanzialmente inalterato.
        </p>
      </div>
    </>
  );
}

export default Home;
