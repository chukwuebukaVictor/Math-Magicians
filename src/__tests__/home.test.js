import { render } from '@testing-library/react';
import Home from '../pages/Home';

it('Checking if Home page renders correctly', () => {
  const tree = render(<Home />);
  expect(tree).toMatchSnapshot();
});
