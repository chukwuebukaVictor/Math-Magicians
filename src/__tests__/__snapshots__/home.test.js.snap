// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Checking if Home page renders correctly 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="welcome"
      >
        <h1>
          Welcome to Our Page
        </h1>
        <p>
          Lorem Ipsum è un testo segnaposto utilizzato nel settore della tipografia e della stampa. Lorem Ipsum è considerato il testo segnaposto standard sin dal sedicesimo secolo, quando un anonimo tipografo prese una cassetta di caratteri e li assemblò per preparare un testo campione. È sopravvissuto non solo a più di cinque secoli, ma anche al passaggio alla videoimpaginazione, pervenendoci sostanzialmente inalterato.
        </p>
        <p>
          Lorem Ipsum è un testo segnaposto utilizzato nel settore della tipografia e della stampa. Lorem Ipsum è considerato il testo segnaposto standard sin dal sedicesimo secolo, quando un anonimo tipografo prese una cassetta di caratteri e li assemblò per preparare un testo campione. È sopravvissuto non solo a più di cinque secoli, ma anche al passaggio alla videoimpaginazione, pervenendoci sostanzialmente inalterato.
        </p>
        <p>
          Lorem Ipsum è un testo segnaposto utilizzato nel settore della tipografia e della stampa. Lorem Ipsum è considerato il testo segnaposto standard sin dal sedicesimo secolo, quando un anonimo tipografo prese una cassetta di caratteri e li assemblò per preparare un testo campione. È sopravvissuto non solo a più di cinque secoli, ma anche al passaggio alla videoimpaginazione, pervenendoci sostanzialmente inalterato.
        </p>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="welcome"
    >
      <h1>
        Welcome to Our Page
      </h1>
      <p>
        Lorem Ipsum è un testo segnaposto utilizzato nel settore della tipografia e della stampa. Lorem Ipsum è considerato il testo segnaposto standard sin dal sedicesimo secolo, quando un anonimo tipografo prese una cassetta di caratteri e li assemblò per preparare un testo campione. È sopravvissuto non solo a più di cinque secoli, ma anche al passaggio alla videoimpaginazione, pervenendoci sostanzialmente inalterato.
      </p>
      <p>
        Lorem Ipsum è un testo segnaposto utilizzato nel settore della tipografia e della stampa. Lorem Ipsum è considerato il testo segnaposto standard sin dal sedicesimo secolo, quando un anonimo tipografo prese una cassetta di caratteri e li assemblò per preparare un testo campione. È sopravvissuto non solo a più di cinque secoli, ma anche al passaggio alla videoimpaginazione, pervenendoci sostanzialmente inalterato.
      </p>
      <p>
        Lorem Ipsum è un testo segnaposto utilizzato nel settore della tipografia e della stampa. Lorem Ipsum è considerato il testo segnaposto standard sin dal sedicesimo secolo, quando un anonimo tipografo prese una cassetta di caratteri e li assemblò per preparare un testo campione. È sopravvissuto non solo a più di cinque secoli, ma anche al passaggio alla videoimpaginazione, pervenendoci sostanzialmente inalterato.
      </p>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;

exports[`Test for Home component The component should render 1`] = `
<DocumentFragment>
  <div
    class="welcome"
  >
    <h1>
      Welcome to Our Page
    </h1>
    <p>
      Lorem Ipsum è un testo segnaposto utilizzato nel settore della tipografia e della stampa. Lorem Ipsum è considerato il testo segnaposto standard sin dal sedicesimo secolo, quando un anonimo tipografo prese una cassetta di caratteri e li assemblò per preparare un testo campione. È sopravvissuto non solo a più di cinque secoli, ma anche al passaggio alla videoimpaginazione, pervenendoci sostanzialmente inalterato.
    </p>
    <p>
      Lorem Ipsum è un testo segnaposto utilizzato nel settore della tipografia e della stampa. Lorem Ipsum è considerato il testo segnaposto standard sin dal sedicesimo secolo, quando un anonimo tipografo prese una cassetta di caratteri e li assemblò per preparare un testo campione. È sopravvissuto non solo a più di cinque secoli, ma anche al passaggio alla videoimpaginazione, pervenendoci sostanzialmente inalterato.
    </p>
    <p>
      Lorem Ipsum è un testo segnaposto utilizzato nel settore della tipografia e della stampa. Lorem Ipsum è considerato il testo segnaposto standard sin dal sedicesimo secolo, quando un anonimo tipografo prese una cassetta di caratteri e li assemblò per preparare un testo campione. È sopravvissuto non solo a più di cinque secoli, ma anche al passaggio alla videoimpaginazione, pervenendoci sostanzialmente inalterato.
    </p>
  </div>
</DocumentFragment>
`;
